Enterprise-CMCS/macpro-mdct-mcr
# [MDCT-2344] Add entity details overlay export page

### Description
This PR adds a few new components (`ExportedEntityDetailsOverlaySection`, `ExportedEntityDetailsTable` and `ExportedEntityDetailsTableRow`) which allow us to render the final page of the MLR pdf export.

These new components are required because the entity details are rendered in several distinct tables grouped by section, rather than one large one. This ended up being enough of a difference to make it worthwhile to create all new components specifically for rendering these table sections.

### Related ticket(s)


MDCT-2344

---
### How to test

1. Create and fill out an MLR submission with at least one report.
2. Navigate to the Review & Submit page
3. Navigate to the final page of the exported report and verify that your report's data appears correctly.


### Important updates


None

---
### Author checklist


---


_convert to a different template: [test → val](?expand=1&template=test-to-val-deployment.md)_ | _[val → prod](?expand=1&template=val-to-prod-deployment.md)_
